tools/mgba-rom-test-hydra/main.c

```diff
@@ -35,7 +36,7 @@
 #define min(a, b) ((a) < (b) ? (a) : (b))
 
 #define MAX_PROCESSES               32 // See also test/test.h
-#define MAX_SUMMARY_TESTS_TO_LIST   50
+#define MAX_SUMMARY_TESTS_TO_LIST   2
```

Comment:
Can this be increased? At least to something like 20 because it's pretty common in fixing that something that suddenly a bunch of stuff fails

Reply:
Oops, this wasn't meant to go through. I'll revert it to 50 as it has been ^^;

Reply:
done